django-ecommerce/celery.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import, unicode_literals
+import os
+from celery import Celery
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'django-ecommerce.settings')
+
+app = Celery('django-ecommerce')
+
+app.config_from_object('django.conf:settings', namespace='CELERY')
+
+app.autodiscover_tasks()
+
+@app.task(bind=True)
+def debug_task(self):
```

Comment:
unnecessary task, production code shouldn't have these sorts of debug tasks or print statements.